docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -86,202 +147,342 @@ kubeconfig using the hypershift cli.
+
+Let's go ahead and show how to configure this option.
+
+1. Enable wildcard routes on the management cluster.
+
+    ```sh
+    oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
+    ```
+
+2. Create HyperShift KubeVirt cluster with the base domain `apps.mgmt-cluster.krnl.es`
+
+    ```shell linenums="1"
+    export CLUSTER_NAME=example
+    export PULL_SECRET="$HOME/pull-secret"
+    export MEM="6Gi"
+    export CPU="2"
+    export WORKER_COUNT="2"
+    export BASE_DOMAIN=apps.mgmt-cluster.krnl.es
+    # OCP_RELEASE is optional, this will determine which version is used for the Hosted Cluster
+    # if not set, it will default to the hardcoded version in the hypershift cli
+    export OCP_RELEASE=4.12.7-x86_64
+
+    hypershift create cluster kubevirt \
+    --name $CLUSTER_NAME \
+    --node-pool-replicas $WORKER_COUNT \
+    --pull-secret $PULL_SECRET \
+    --memory $MEM \
+    --cores $CPU \
+    #--release-image quay.io/openshift-release-dev/ocp-release:$OCP_RELEASE \
+    --base-domain $BASE_DOMAIN
+    ```
+
+3. Get the kubeconfig
+
+    ```shell
+    hypershift create kubeconfig --name $CLUSTER_NAME > $CLUSTER_NAME-kubeconfig
+    ```
+
+4. Create a ClusterIP service and a OCP Route in the management cluster for providing ingress to the hosted cluster (One limitation is that we can only expose one of the router ports, in this case we will expose the https one.)
+
+    ```shell
+    export HTTPS_NODEPORT=$(oc --kubeconfig $CLUSTER_NAME-kubeconfig get services -n openshift-ingress router-nodeport-default -o jsonpath='{.spec.ports[?(@.name=="https")].nodePort}')
+    cat << EOF | oc apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      labels:
+        app: $CLUSTER_NAME
+      name: $CLUSTER_NAME-apps
+      namespace: clusters-$CLUSTER_NAME
+    spec:
+      ports:
+      - name: https-443
+        port: 443
+        protocol: TCP
+        targetPort: ${HTTPS_NODEPORT}
+      selector:
+        kubevirt.io: virt-launcher
+      type: ClusterIP
+    EOF
+    cat << EOF | oc apply -f -
+    apiVersion: route.openshift.io/v1
+    kind: Route
+    metadata:
+      name: $CLUSTER_NAME-443
+      namespace: clusters-$CLUSTER_NAME
+    spec:
+      host: data.apps.$CLUSTER_NAME.$BASE_DOMAIN
+      wildcardPolicy: Subdomain
+      tls:
+        termination: passthrough
+        insecureEdgeTerminationPolicy: Redirect
+      port:
+        targetPort: https-443
+      to:
+        kind: Service
+        name: $CLUSTER_NAME-apps
+        weight: 100
+    EOF
+    ```
+
+## Checking HostedCluster status after having fixed the ingress
+
+Now that we fixed the ingress, we should see our HostedCluster progress moved from `Partial` to `Completed`.
```

Comment:
this is done automatically now for the latest hypershift operator as long as guest clsuters are using 4.12.2 or greater

Reply:
Correct, removed.